src/services/export.service.js

```diff
@@ -67,6 +67,46 @@ const patientBulkExport = async (request, reply) => {
   }
 };
 
+/**
+ * Exports data from a FHIR server for resource types pertaining to patients found in the referenced
+ * Group. Uses parsed patient compartment definition similarly to patientBulkExport.
+ * @param {Object} request the request object passed in by the user
+ * @param {Object} reply the response object
+ */
+const groupBulkExport = async (request, reply) => {
+  if (validateExportParams(request, reply)) {
+    request.log.info('Group >>> $export');
+    const group = await findResourceById(request.params.groupId, 'Group');
```

Comment:
When testing, I tried sending a GET `[base]/Group/[id]/$export` request for a Group ID that does not exist on the server. I got a 500 Internal Server Error "cannot read property 'member' of null." I think what's happening is the `group` variable here becomes null and so it errors out when doing the mapping on the group members. Adding a 404 error when the group is null should prevent this 500 from being thrown